src/mono/wasm/build/WasmApp.targets

```diff
@@ -199,6 +199,11 @@
   </Target>
 
   <Target Name="_BeforeWasmBuildApp" DependsOnTargets="_SetupEmscripten;_SetWasmBuildNativeDefaults">
+    <PropertyGroup>
+      <MicrosoftNetCoreAppRuntimePackDir Condition="'$(MicrosoftNetCoreAppRuntimePackDir)' == ''">$([MSBuild]::NormalizeDirectory($(NuGetPackageRoot), 'microsoft.netcore.app.runtime.browser-wasm', $(BundledNETCoreAppPackageVersion)'))</MicrosoftNetCoreAppRuntimePackDir>
```

Comment:
```suggestion
      <MicrosoftNetCoreAppRuntimePackDir Condition="'$(MicrosoftNetCoreAppRuntimePackDir)' == ''">$([MSBuild]::NormalizeDirectory($(NuGetPackageRoot), 'microsoft.netcore.app.runtime.mono.browser-wasm', $(BundledNETCoreAppPackageVersion)'))</MicrosoftNetCoreAppRuntimePackDir>
```

Reply:
the name changed only for the package id

Reply:
https://github.com/dotnet/sdk/blob/main/src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.Current.targets#L75

Reply:
Lol my bad! We are using that nuget package path, so that would be the new name!